users/clients/api/endpoints.go

```
@@ -129,6 +151,9 @@ func updateClientEndpoint(svc clients.Service) endpoint.Endpoint {
 		}
 		client, err := svc.UpdateClient(ctx, req.token, client)
 		if err != nil {
+			// if !errors.Contains(err, apiutil.ErrValidation) {
```

empty block, do not mask error